src/components/taxonomicService/taxonomicServiceFormComponents/FormBuilder.tsx

```diff
@@ -142,6 +143,13 @@ const FormBuilder = (props: Props) => {
                 return <MultiSelectField field={field}
                     SetFieldValue={(fieldName: string, value: string) => SetFieldValue?.(fieldName, value)}
                 />;
+            } case 'ror': {
```

Comment:
maybe some exception handling? if the organisation isn't found?

Reply:
It does say no options available when there is no hit within ROR.
Additional help on this like an exception is a good improvement for the future.